### PR TITLE
build: release: v1.13.2-rc4

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
 # Lotus changelog
 
-# v1.13.2-rc3 /  2021-12-09
+# v1.13.2-rc4 /  2021-12-10
 
-This is the second RC for lotus v1.13.2, with a dependency upgrade of ipld-legacy which will fix ChainGetNode. This 
-is a highly recommended release with sealing pipeline fixes, worker management and scheduler enhancement, retrieval improvements and so on. More detailed changelog will be updated later.
+This is the 4th RC for lotus v1.13.2, with another retrieval enhancement that fills the gap that;s brought by the release. This is a highly recommended release with sealing pipeline fixes, worker management and scheduler enhancement, retrieval improvements and so on. More detailed changelog will be updated later.
```
Comment (arajasek) [inline CHANGELOG.md:5]:
```suggestion
This is the 4th RC for lotus v1.13.2, with another retrieval enhancement that fills the gap that's brought by the release. This is a highly recommended release with sealing pipeline fixes, worker management and scheduler enhancement, retrieval improvements and so on. More detailed changelog will be updated later.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
 - github.com/filecoin-project/lotus:
-  - CARv2 v2.1.0
-  - dagstore pieceReader: Fix wrong ErrUnexpectedEOF return in ReadAt
-  - dagstore pieceReader: Always read full in ReadAt
-  - Add metrics to dagstore piecereader
-  - disable building of appimage on release
+  - stores: Reduce log spam during retrievals
+  - fix lint
```
Comment (arajasek) [inline CHANGELOG.md:9]: would remove fix lint from changelog

**Review by arajasek — APPROVED** [review]

**Comment by stuberman** [comment]
If upgrading from v1.13.2-rc3 do we only need to update market node? Or do we need to also update miner or workers?

File: CHANGELOG.md (hunk 2 above)
Comment (jennijuju) [reply CHANGELOG.md:9]: this is a cherry-picked commit   - in the final changelog it will be updated with the pr!

**Comment by jennijuju** [comment]
> If upgrading from v1.13.2-rc3 do we only need to update the market node? Or do we need to also update miners or workers?

AFAIK, it needs to add miner/worker too as it changes how to read from the sector storage.

**Pull request merged**